docs/FAQs.md

```diff
@@ -13,6 +13,131 @@ After you set up a repository, you can substitute in any tools you'd like.
 
 If you think the tool would be broadly useful to most consumers of this template, feel free to [file a feature request](https://github.com/JoshuaKGoldberg/create-typescript-app/issues/new?assignees=&labels=type%3A+feature&projects=&template=03-feature.yml&title=%F0%9F%9A%80+Feature%3A+%3Cshort+description+of+the+feature%3E) to add it in.
 
+## Can I create a GitHub action with CTA?
+
+Yes! If you want to read in detail about GitHub Actions [try this](https://docs.github.com/en/actions/creating-actions).
+Here we'll outline the steps required to migrate a CTA app to a GitHub Action:
+
+1. The GitHub Actions does not cater for tabs well, so you will likely want to set your `README.md` to use spaces, not tabs.
```

Comment:
[Question] I know we chatted about this annoyance a bit but I didn't realize it's a blocker. What is it GHA doesn't do well, specifically?

https://github.com/JoshuaKGoldberg/all-contributors-auto-action uses tabs and is fine. Since YML doesn't support tabs _(UGH!)_ the <code>```yml</code> code blocks and `.y(a)ml` files there are spaces still.

Reply:
The yaml code gets weird wonky highlighting. It's possible that the non yaml text is fine?

Reply:
Either way, the confusion that arises from mixing tabs and spaces in a single file feels like something worth avoiding.

Reply:
Do you have an example of the wonkiness? I haven't found anything.

Reply:
Sure - look at the yaml in the readme of the initial release if Schemar: https://github.com/marketplace/actions/schemar-ci-action?version=v0.1.0

Reply:
Interesting. That's mixed spaces and tabs - otherwise there'd be red covering all indentations.

Running `pnpm format --write` doesn't change it. But if I manually replace the tabs with spaces, Prettier still is happy. No exit codes either way.

So I think the resolution here should be to not mix spaces and tabs?

Reply:
Removed the tabs / spaces stuff